Documentation/typo3cms/GuidesAndTutorials/Index.rst

```diff
@@ -77,18 +103,29 @@ represents a logical progression in the knowledge of the product.
  :header-rows: 1
```

Comment:
Should we move the tutorials before the guides?
They are more important to newcomers. Experienced users might find the guides even if they are more "hidden".

Reply:
@DanielSiepmann I agree to move the Tutorials before the Guides. I had the same thought. I think, someone else already suggested the same thing (can't remember who it was).

Thank you for your help!

I would suggest merging first and then to make further changes, but it seems you are doing it that way already with the other PR.

Reply:
I've pushed the change.